docs/readmes/README.msi.md

```diff
@@ -48,6 +48,10 @@ Now to ensure that the operations are allowed on individual identity, perform th
 ```bash
 az role assignment create --role "Managed Identity Operator" --assignee <principal id from above command>  --scope /subscriptions/<subscription id>/resourcegroups/<resource group name>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<identity name>
 ```
+Note: If the VNET is located outside of the AKS resource group, the following command will need to be run on the VNET resource group as well.
+```bash
+az role assignment create --role "Virtual Machine Contributor" --assignee <principal id from above command>  --scope /subscriptions/<sub id>/resourcegroups/<resource group name>
```

Comment:
Does it need it to be `Virtual Machine Contributor` role or can it just be `Network Contributor`? Referring the [docs](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal#networking)

Reply:
This is the right that it has an issue with:

Microsoft.Network/virtualNetworks/subnets/join/action

As you stated, this link will help ensure that the managed identity has the proper permissions.

https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal#networking